app/Console/Commands/GoogleSync.php

```diff
@@ -158,6 +159,98 @@ public function syncGoogleGroupsWithCommittees(): void
         }
     }
 
+    /**
+     * Synchronise the current Proto aliases with the Google Workspace groups.
+     *
+     *
+     * @throws Throwable
+     */
+    public function syncGoogleGroupsWithAliases(): void
+    {
+        $this->output->info('Aliases:');
+
+        $aliases = ProtoAlias::query()->groupBy('alias')->get();
+        $googleGroups = $this->listGoogleGroups();
+        $googleAliasGroups = $googleGroups->filter(fn ($group) => Str::startsWith($group->name, 'Alias'));
+
+        $groupsToAdd = $aliases->reject(fn ($alias): bool => $googleAliasGroups->contains('email', $alias->alias));
+        $groupsToRemove = $googleAliasGroups->reject(fn ($group): bool => $aliases->contains('alias', $group->email));
+
+        foreach ($groupsToRemove as $group) {
+            $this->pp(
+                '<fg=red>-</> '.str_pad("#$group->id", 6).str_pad("$group->name <fg=gray>", 65, '.')."</> $group->email",
+                fn () => $this->directory->groups->delete($group->id)
+            );
+        }
+
+        foreach ($groupsToAdd as $alias) {
+            try {
+                $this->pp(
+                    '<fg=yellow>+</> '.str_pad("#$alias->id", 6).$alias->alias,
+                    fn () => $this->directory->groups->insert(
+                        new GoogleGroup([
+                            'name' => "Alias $alias->alias",
+                            'email' => $alias->alias,
+                            'description' => 'Email alias '.$alias->alias,
+                        ])
+                    )
+                );
+            } catch (Throwable $throwable) {
+                $this->pp(
+                    '<fg=red>x</> '.str_pad("#$alias->id", 6).$alias->alias,
+                    fn (): mixed => dump($throwable->getMessage())
+                );
+            }
+        }
+    }
+
+    /**
+     * Synchronise the current Proto alias members with the Google Workspace group members.
+     *
+     *
+     * @throws Throwable
+     */
+    public function syncGoogleGroupMembersWithExternalAliasesMembers(): void
+    {
+        $this->output->info('Alias Members:');
+
+        $aliases = ProtoAlias::query()->whereNotNull('destination')->get()->groupBy('alias');
+        $googleGroups = $this->listGoogleGroups();
+        $googleAliasGroups = $googleGroups->filter(fn ($group) => Str::startsWith($group->name, 'Alias'));
+
+        foreach ($aliases as $aliasGroup) {
+            $this->pp(
+                '<fg=green>✓</> '.str_pad("$aliasGroup->first()->alias <fg=gray>", 65, '.')."</> $aliasGroup->count() members",
```

Comment:
This print does not work, you can only acces the variable in this way and not the children.